tests/unit/components/cylc/tree/tree.data.js

```diff
@@ -7,7 +7,7 @@ const simpleWorkflowTree4Nodes = [
     __type: 'workflow',
     id: 'user/workflow1',
     name: 'workflow1',
-    status: 'running',
+    state: 'running',
```

Comment:
Need to standardise the use of `state` and `status` throughout Cylc.

From `cylc.flow.task_state` "state" is a combination of the following:

```
    __slots__ = [
        "external_triggers",
        "is_held",
        "identity",
        "is_updated",
        "kill_failed",
        "outputs",
        "prerequisites",
        "status",
        "suicide_prerequisites",
        "time_updated",
        "xtriggers",
        "_is_satisfied",
        "_suicide_is_satisfied",
    ]
```

(most importantly "status" & "is_held")

Reply:
Agreed. Always find it weird when I type `task.status = node.state`. I think the issue is that we are using `status` in the UI, but the data for the status is in the `.state` attribute in GraphQL entities like `TaskProxy`, `Job`, etc.

Do we need an issue in cylc-admin for that, or maybe in Cylc UI Server?

Reply:
Probably my fault, way back - sorry!

Reply:
Did anyone post an issue already?

Reply:
> Did anyone post an issue already?

I didn't post one yet, and also did not see any issue for this on my notifications.

Reply:
I'll bang up a quick one now...

Reply:
Issue posted on cylc-flow (that's were the problem starts, even it it goes further)

Reply:
Thanks!! :tada: